### PR TITLE
Add Handle#isReleased()

### PR DESCRIPTION
**Pull request by verticalpalette**

### CODE REVIEW

**Comment by pspeter3** [comment]
Can you push a new version?

``` sh
npm version patch
git push origin master --follow-tags
```

File: src/perishable.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ class PerishableNode<T> implements Handle<T> {
     _callback: () => any;
     _prev: PerishableNode<T>;
     _next: PerishableNode<T>;
+    _is_released: boolean;
```
Comment (vsiao) [inline src/perishable.ts:8]: should these all be `private`?

Comment (pspeter3) [inline src/perishable.ts:8]: They're package private basically.

Comment (vsiao) [inline src/perishable.ts:8]: I don't know what that means.

Comment (pspeter3) [inline src/perishable.ts:8]: It's not publicly exported and is only used by the Perishable class. If it was private, the perishable class would not be able to do as much with it.

Comment (vsiao) [inline src/perishable.ts:8]: Oh, I didn't realize that this was a private class being used by `Perishable`.

**Pull request merged**